specificatie/openapi.yaml

```diff
@@ -1391,33 +1276,24 @@ components:
               \ 'gemeentecode' (volgens GBA tabel 33, Gemeententabel), de tweecijferige\
               \ 'objecttypecode' en een voor het betreffende objecttype binnen een gemeente\
               \ uniek tiencijferig 'objectvolgnummer'."
-            maxLength: 16
             example: "0518200000366054"
           indicatieVestigingVanuitBuitenland:
             type: boolean
-            title: "indicatieVestigingVanuitBuitenland"
             description: "Indicatie waarmee aangegeven wordt of de ingeschreven persoon\
               \ zich vanuit het buitenland heeftingeschreven. Deze indicatie heeft als\
               \ rol om aan te geven dat iemand zich vanuit het buitenland gevestigd\
               \ heeft. Deze indicator wordt altijd meegeleverd als de waarde true is.\
               \ Als de waarde false is wordt de indicator niet meegeleverd."
           locatiebeschrijving:
             type: "string"
-            title: "Locatiebeschrijving"
             description: "Een geheel of gedeeltelijke omschrijving van de ligging van\
               \ een object."
-            maxLength: 35
             example: "Naast de derde brug"
-          straatnaam:
-            type: "string"
-            title: "Straatnaam"
-            description: "De officiële straatnaam zoals door het bevoegd gemeentelijk\
-              \ orgaan is vastgesteld, zo nodig ingekort conform de specificaties van\
-              \ de NEN 5825. alle alfanumrieke tekens"
-            maxLength: 24
+          korteNaam:
```

Comment:
Voegen we die ook toe? Daar hebben we geen feature voor. Nice to have, die we altijd nog kunnen toevoegen op een later moment.

Reply:
Deze is toegevoegd vanwege het gelijktrekken van naamgeving met de BAG. In de BRP was "straatnaam" maximaal 24 karakters lang. Dat heet in de BAG de korteNaam.

Reply:
@CathyDingemanse wil je naast straat(naam) ook kortenaam in verblijfplaats? In BRP is straatnaam al maximaal 24 tekens lang, dus korteNaam zal dan altijd dezelfde waarde geven als straat (naamOpenbareRuimte kan wel anders/langer zijn, max. 80 tekens).

Reply:
Ik heb voor deze discussie #659 aangemaakt. Laten we daar de discussie verder voeren